biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.css

```diff
@@ -0,0 +1,18 @@
+.simulation {
+  background-color: #f2f2f2;
```

Comment:
nit: this works for now, but to make it easier, it might be helpful to have some css constants class to define things like your app background and app font, and pull that into every page and read from there

Reply:
Nice will check it out